coremark_pro/coremark_pro_run

```diff
@@ -262,6 +276,9 @@ run_coremark_pro()
 		numb_cpus=`cat /proc/cpuinfo | grep processor | wc -l`
 		for iter in $(seq 1 1 $to_times_to_run); do
 			make TARGET=linux64 XCMD=-c${numb_cpus} certify-all > coremark_pro_run_${iter}
+			if [ $? -ne 0 ]; then
+				exit_out "make Failed, make TARGET=linux64 XCMD=-c${numb_cpus} certify-all" 1
```

Comment:
Consider putting the make args into a variable, that way (in the unlikely event) of the make args changing, you only need to update in one place